staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -255,13 +255,13 @@ func recordEvent(sink EventSink, event *v1.Event, patch []byte, updateExistingEv
 	switch err.(type) {
 	case *restclient.RequestConstructionError:
 		// We will construct the request the same next time, so don't keep trying.
-		klog.Errorf("Unable to construct event '%#v': '%v' (will not retry!)", event, err)
+		klog.Errorf("Unable to construct event '%#v, Message: %s': '%v' (will not retry!)", event.InvolvedObject, event.Message, err)
 		return true
 	case *errors.StatusError:
 		if errors.IsAlreadyExists(err) {
-			klog.V(5).Infof("Server rejected event '%#v': '%v' (will not retry!)", event, err)
+			klog.V(5).Infof("Server rejected event '%#v, Message: %s': '%v' (will not retry!)", event.InvolvedObject, event.Message, err)
 		} else {
-			klog.Errorf("Server rejected event '%#v': '%v' (will not retry!)", event, err)
+			klog.Errorf("Server rejected event '%#v, Message: %s': '%v' (will not retry!)", event.InvolvedObject, event.Message, err)
```

Comment:
This change looks reasonable, but I'd prefer to see something like

```go
if klog.V(4){
  // show everything
  klog.Errorf("show everything")
} else{
  klog.Errorf("show less than everything")
}
```

Reply:
I think it should be optimized.

Reply:
it is kube-scheduler.
the log:
`E0821 06:35:49.607585   98018 event.go:263] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"multiimage-app-test-0-multi-image-service-ms-2-77564df8d8-l4mr5.162d1ac7e13c9cd4", GenerateName:"", Namespace:"ststable", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Pod", Namespace:"ststable", Name:"multii-test-0-multii-service-ms-2-77234df8d8-l4mr5", UID:"094c963d-10e5-4249-bf93-3312dbe3873c", APIVersion:"v1", ResourceVersion:"111571", FieldPath:""}, Reason:"Scheduled", Message:"Successfully assigned ststable/multiimage-app-test-0-multi-image-service-ms-2-77564df8d8-l4mr5 to 172.20.0.10", Source:v1.EventSource{Component:"default-scheduler", Host:""}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xbfc7dc8221bd4cd4, ext:32296177949085, loc:(*time.Location)(0x2b88a00)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xbfc7dc8221bd4cd4, ext:32296177949085, loc:(*time.Location)(0x2b88a00)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'Too many requests: limit reached on type User for key kubecfg' (will not retry!)`